linode/manager
# M3-5503: Add script to increment package versions during release

## Description

**What does this PR do?**
This adds a small script to increment our package versions during the release process. It can be executed like this:

```
yarn package-versions
```
You'll be prompted to specify a version for each package. Alternatively, if you already know which version each package should be, you can invoke it non-interactively like this:

```
yarn package-versions <manager-version> <api-version> <validation-version> -f
```

In either case, a summary will be displayed which reflects the changes made by the script.

Because we now use `*` to specify workspace package dependency versions, no changes get made to the package `dependencies` fields.

This probably could have been done more concisely or with a fancier UI, but I wanted the script to run without any additional dependencies so that running `yarn install` beforehand would not be necessary.

## How to test

**What are the steps to reproduce the issue or verify the changes?**
Run the command with a variety of option/flag combinations to ensure that the behavior works as expected:

Runs the script interactively:
```
yarn package-versions
```

Runs the script interactively, with Cloud Manager `v1.75.0`, API v4 `v0.77.0`, and Validation `v0.14.0` specified by default:
```
yarn package-versions 1.75.0 0.77.0 0.14.0
```

Runs the script non-interactively, updating Cloud Manager to `v1.75.0`, APIv4 to `v0.77.0`, and Validation to `v0.14.0`:
```
yarn package-versions 1.75.0 0.77.0 0.14.0 -f
```

Runs the script non-interactively, only specify Cloud Manager `v1.75.0`, which should result in an error:
```
yarn package-versions 1.75.0 -f
```

Running the script will modify your `package.json` files, so you'll want to discard those changes afterwards.
